instrumentation/instagocb/instagocb.go

```diff
@@ -0,0 +1,194 @@
+// (c) Copyright IBM Corp. 2023
+
```

Comment:
```suggestion
// Package Instagocb provides instrumentation for the gocb package
```

Add a line with the package information so that it appears in the godoc